Bluejay.asm

```diff
@@ -3754,7 +3800,11 @@ input_high_check:
 
 	call	beep_enter_bootloader
 
-	ljmp	1C00h					; Jump to bootloader
+	IF MCU_48MHZ == 2
+		ljmp	0F000h				; Jump to bootloader
```

Comment:
Yes, it is outside of the flash, but it the address intended for the bootloader. There is a page reserved for it. If the bootloader  is bigger then one page, then the last page of Flash should be used. I mean, technically it makes no real difference, but since BLHeli has it's bootloader there, I would really like to keep it this way in order to not have to handle some strange edge cases...

Reply:
Agreed